config/accounts.go

```diff
@@ -255,11 +256,12 @@ func (bidderCodes *AlternateBidderCodes) IsValidBidderCode(bidder, alternateBidd
 		return false, fmt.Errorf(ErrAlternateBidderNotDefined, bidder, alternateBidder)
 	}
 
-	if len(adapterCfg.AllowedBidderCodes) == 0 {
+	if !adapterCfg.Enabled {
+		// config has bidder entry but is not enabled, report it
 		return false, fmt.Errorf("alternateBidderCodes disabled for %q, rejecting bids for %q", bidder, alternateBidder)
 	}
 
-	if adapterCfg.AllowedBidderCodes[0] == "*" {
+	if len(adapterCfg.AllowedBidderCodes) == 0 || adapterCfg.AllowedBidderCodes[0] == "*" {
```

Comment:
I think this would defeat the purpose of bidder level ` alternatebiddercodes.bidders.BIDDER.enabled` flag. User can simply set it false to disable and true to enable. Just that empty array would be one of the corner case under bidder level flag enabled. Wonder, why would user set bidder level flag enabled and then define empty array to reject all the bidder codes instead of simply setting the bidder level flag false.

Reply:
Update the code. I hope this behaviour is similar in PBJS aswell.

Reply:
I left a comment here https://github.com/prebid/Prebid.js/pull/8760#issuecomment-1201576945 to ensure that the behavior is the same for both.

Reply:
Catching on this thread, I disagree with @bsardo on this point. I prefer the original implementation where an empty list is treated the same as a nil list. Otherwise, I believe we're leaning too much on the config system for how to represent the two concepts. I could easily forsee one account fetcher returning nil and another returning empty for the same concept.

Reply:
@pm-nilesh-chate please hold while we figure out what to do here 🙂

Reply:
Sure, I'll wait until this discussion is concluded. Whatever the outcome is, I would prefer to have similar behaviour in both the PBS and PBJS (fyi, PBJS change has been merged https://github.com/prebid/Prebid.js/pull/8760#issuecomment-1201576945).

Reply:
We'll leave it as is, especially since PBJS already made the same change.